### DISCUSSION TITLE
[视频] OpenSSF 系列视频若干

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#68] [资料分享]
**Learn about the new tool Scorecard**
- https://www.youtube.com/watch?v=emLqpvPdsGs

**Introduction to Project Alpha-Omega**
- https://www.youtube.com/watch?v=RBVzDGi66aw

**OpenSSF APAC Wipro**
- https://www.youtube.com/watch?v=byWJOfQXRG4

<img src="https://user-images.githubusercontent.com/15010826/161706022-c81366c7-8756-429b-b1ab-77924d2e8dfb.png" width="500px">

**Comment by wangleo61** [comment]
太给了

**Comment by wangleo61** [comment]
哎呦，给力呀，王老师，货真多

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Tuesday, April 5, 2022 3:39 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] [视频] OpenSSF 系列视频若干 (Discussion #68)


Learn about the new tool Scorecard
https://www.youtube.com/watch?v=emLqpvPdsGs

Introduction to Project Alpha-Omega

  *   https://www.youtube.com/watch?v=RBVzDGi66aw

OpenSSF APAC Wipro

  *   https://www.youtube.com/watch?v=byWJOfQXRG4

―
